config/defaults.go

```diff
@@ -0,0 +1,82 @@
+package config
+
+import (
+	"net"
+	"time"
+
+	"github.com/bitcoin-sv/pulse/domains/logging"
+)
+
+// DBSqlite creating config for sqlite db.
+const DBSqlite DbType = "sqlite"
+
+func GetDefaultAppConfig(lf logging.LoggerFactory) *AppConfig {
+	return &AppConfig{
+		DbConfig:         getDbDefaults(),
+		HTTPConfig:       getAuthConfigDefaults(),
+		MerkleRootConfig: getMerkleRootDefaults(),
+		WebsocketConfig:  getWebsocketDefaults(),
+		WebhookConfig:    getWebhookDefaults(),
+		P2PConfig:        getP2PDefaults(lf),
+	}
+}
+
+func getDbDefaults() *DbConfig {
+	return &DbConfig{
+		Type:               DBSqlite,
+		FilePath:           "./data/blockheaders.db",
+		Dsn:                "file:./data/blockheaders.db?_foreign_keys=true&pooling=true",
+		SchemaPath:         "./database/migrations",
+		PreparedDb:         false,
+		PreparedDbFilePath: "./data/blockheaders.csv.gz",
+	}
+}
+
+func getAuthConfigDefaults() *HTTPConfig {
```

Comment:
Name can be misleading. Maybe just getHttpConfigDefaults?

Reply:
done